### PR TITLE
Enable auto-yasnippet for company

### PR DESCRIPTION
**Pull request by gbastkowski**
In https://github.com/syl20bnr/spacemacs/pull/16176 the auto-yasnippet package has been toggled off for company auto-completion. I am not sure if that was really intended. I would like to have it available.

### CODE REVIEW

**Comment by bcc32** [comment]
This PR changes auto-yasnippet to be toggled on _only for company_ and not if company is disabled.  Is that intentional?

**Comment by gbastkowski** [comment]
No, it was not.

File: layers/+completion/auto-completion/packages.el [hunk 1]
```diff
@@ -91,6 +91,7 @@
     (spacemacs/set-leader-keys
       "iSc" 'aya-create
       "iSe" 'spacemacs/auto-yasnippet-expand
+      "iSh" 'spacemacs/auto-yasnippet-expand-from-history
```
Comment (smile13241324) [inline layers/+completion/auto-completion/packages.el:94]: I think you should add your new binding into the layer readme.

**Review by smile13241324 — CHANGES_REQUESTED** [review]
I have found a small issue please have a look and ping me when you are ready for another review.

**Review by smile13241324 — APPROVED** [review]

**Pull request merged**